lib/jobmanager.py

```diff
@@ -275,6 +275,33 @@ async def _cancel_jobs_for_series(cls, series_name: str):
                 await cls._update_queue_cb(
                     SUBSCRIPTION_CHANGE_TYPE_DELETE, job.rid)
 
+    @classmethod
+    @cls_lock()
+    async def cancel_jobs_by_config_name(cls, series_name: str,
```

Comment:
I think this method does not require async?

Reply:
@joente It does, because of the cls_lock decorator

Reply:
This will be refactored in the development branch together with other async fixes, but for now it has to be like this